### PR TITLE
fixing bugs to restore my sanity

### LINKED ISSUE #9960: [BUG] Player sounds are stationary and do not follow the player.
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

23070821

### I was

Client

### OS

Windows 8, 10, 11

### What happened?

If for example, you do a *scream you can "outrun" the sound and it will get quieter the farther you get from it while it is playing.

### What should have happened instead?

Sounds coming from the player should follow the player so they do not become quieter if they quickly move away from it.

### Steps to reproduce the issue

1. Scream
2. Run away from where you were as the sound is playing
3. It should get quieter (easier to tell if you are faster)

### Additional context

_No response_

### LINKED ISSUE #9961: [BUG] Jukebox play 2 songs at once when autoplaying to next song.
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

23071201

### I was

Client

### OS

Windows 8, 10, 11

### What happened?

When you turn on the Jukebox it will play the song you select just fine but then when it auto plays to the next song it will play two songs at once. I believe it plays the one immediately following the song just played and the one after that as well. You can fix it pretty easily by just stopping and starting the song again.

### What should have happened instead?

The Jukebox should only play one song at a time.

### Steps to reproduce the issue

1. Turn on Jukebox
2. Wait for the first song to end and for the next to begin
3. BUG!!!

### Additional context

_No response_

### LINKED ISSUE #9968: [BUG] Vending Machines do not visually update the amount of stock they have.
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

23071201

### I was

Client

### OS

Windows 8, 10, 11

### What happened?

When for example getting snacks from a snack vending machine the amount of a snack a vending machine does not update. It can make it hard to tell what there is actually stock of or not.
https://github.com/unitystation/unitystation/assets/119550415/4ecc760e-7f8f-4826-a49a-ecf148c93ae5

### What should have happened instead?

The vending machine's ui should be consistent with the amount of stock of an item it has.

### Steps to reproduce the issue

1. Go to a vending machine
2. Buy something from it

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MaxIsJoe**
# Linked Issues

Fix #9968
Fix #9961
Fix #9960

# Media

image

# Changelog

CL: [Improvement] Added an indicator for jukeboxes when they're not supposed to be working due to heavy damage.
CL: [Improvement] Updated the vendor UI to be popped out rather than stuck on the top right corner.
CL: [Improvement] Vendors will have their lights turn off when not powered
CL: [Fix] Fixed vendors disposing items when they shouldn't due to them being disconnected.
CL: [Fix] Fixed Jukeboxes playing audio twice under some circumstances.
CL: [Fix] Fixed a couple of NREs on startup inside the editor.
CL: [Fix] Emote sounds are not attached to players, meaning that they'll follow them now.

### CODE REVIEW

**Comment by corp-0** [comment]
you left quick load enabled, so that's why tests are failing

**Comment by corp-0** [comment]
rip DocsReminder action. Seems like something updated and it is broken now.

**Comment by eurtod** [comment]
thank you max, looking forward to seeing the new vending machines in game

File: UnityProject/Assets/Scripts/Objects/Doors/ConstructibleDoor.cs [hunk 1]
```diff
@@ -102,14 +102,12 @@ public void ServerPerformInteraction(HandApply interaction)
...
 					Chat.AddActionMsgToChat(interaction.Performer,
-						$"You unscrew the {gameObject.ExpensiveName()}'s cable panel.",
 						$"{interaction.Performer.ExpensiveName()} unscrews {gameObject.ExpensiveName()}'s cable panel.");
 				}
 				else
 				{
 					DoorAnimatorV2.RemovePanelOverlay();
 					Chat.AddActionMsgToChat(interaction.Performer,
-						$"You screw in the {gameObject.ExpensiveName()}'s cable panel.",
 						$"{interaction.Performer.ExpensiveName()} screws in {gameObject.ExpensiveName()}'s cable panel.");
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Doors/ConstructibleDoor.cs:111]: no You message?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Objects/Doors/ConstructibleDoor.cs:111]: > [you don't really need to have identical text all the time](https://github.com/unitystation/unitystation/pull/9974/commits/b3b8553c197c6da15dab2525fe0e1a4eba2d8e9b)

**Review by corp-0 — APPROVED** [review]

**Pull request merged**